<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TGitMainWindow</class>
 <widget class="QMainWindow" name="TGitMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>History</string>
  </property>
  <widget class="QSplitter" name="DiffSplitter">
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="CentralSplitter">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QTableView" name="LogView">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
    <widget class="QSplitter" name="DetailsSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="CommitDetailsWidget" name="CommitDetails" native="true"/>
     <widget class="QTableView" name="DiffOverview">
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="FileDiffWidget" name="FileDiff">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="OpenRepositoryAction"/>
   </widget>
   <widget class="QMenu" name="ViewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="ViewMenu"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <widget class="QDockWidget" name="ReferencesDockWidget">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>&amp;References</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QTreeView" name="ReferencesView"/>
  </widget>
  <action name="OpenRepositoryAction">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="CopyHashAction">
   <property name="text">
    <string>Copy Hash</string>
   </property>
  </action>
  <action name="CopyShortHashAction">
   <property name="text">
    <string>Copy Short Hash</string>
   </property>
  </action>
  <action name="CopySummaryAction">
   <property name="text">
    <string>Copy Summary</string>
   </property>
  </action>
  <action name="CopyMessageAction">
   <property name="text">
    <string>Copy Message</string>
   </property>
  </action>
  <action name="actionReferences">
   <property name="text">
    <string>References</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommitDetailsWidget</class>
   <extends>QWidget</extends>
   <header>CommitDetailsWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>FileDiffWidget</class>
   <extends>QSplitter</extends>
   <header>FileDiffWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
